site/docs/flink.md

```diff
@@ -97,15 +97,20 @@ CREATE CATALOG hive_catalog WITH (
   'catalog-type'='hive',
   'uri'='thrift://localhost:9083',
   'clients'='5',
-  'property-version'='1'
+  'property-version'='1',
```

Comment:
(maybe it should be discussed elsewhere) we don't need to tell users configure this `property-version`

Reply:
Seems we could remove it from the example,  it's not a necessary property ( The options description has mentioned that).